src/auction.js

```diff
@@ -155,29 +155,21 @@ export function newAuction({adUnits, adUnitCodes, callback, cbTimeout, labels})
     }
   }
 
-  function done(bidRequestId) {
-    const innerBidRequestId = bidRequestId;
-    return delayExecution(function() {
-      const request = find(_bidderRequests, (bidRequest) => {
-        return innerBidRequestId === bidRequest.bidderRequestId;
-      });
-
-      // this is done for cache-enabled video bids in tryAddVideoBid, after the cache is stored
-      request.doneCbCallCount += 1;
-      bidsBackAll();
-    }, 1);
+  function done(bidderCount) {
+    let doneCalled = 0;
+    return function() {
+      doneCalled++;
+      if (doneCalled === bidderCount) {
```

Comment:
are we concerned that some bidders will call multiple `done` callbacks causing the auction to close early?